api/crossref/permissions.py

```diff
@@ -22,6 +25,10 @@ def has_permission(self, request, view):
             ),
             digestmod=hashlib.sha256,
         ).hexdigest()
-        if signature != data.get('signature', None):
-            raise PermissionDenied('Invalid headers on incoming mail')
+        if 'signature' not in data:
+            exc = 'Signature required in request body'
```

Comment:
Minor: I would expect `exc` to be an exception, by convention, but it's not. How about `error_message`?